oauth2_http/java/com/google/auth/oauth2/ServiceAccountCredentials.java

```diff
@@ -562,6 +583,10 @@ public final URI getTokenServerUri() {
     return tokenServerUri;
   }
 
+  public final int getLifetime() {
```

Comment:
Ideally all time entries come with units. As a last resort, make it clear what the units are. We should probably make all the getters/setters/variables something like `lifetimeInSeconds`

Reply:
Bah, in ImpersonatedCredentials this is already `lifetime`

Reply:
Other credential types don't expose this getter - let's not do it unless necessary.

Reply:
now made it default visibility so it can be used in the unit test